Felix45/StackOverFlowLite
# [finishes #162657578] changed all static links and altered logo

**What does this pull request do**
- Alters all links that refer to static html pages

**Description of the task to be completed**
- Alters all links that refer to static html pages

**How should this be manually tested**

- Clone or fork the repository 

- checkout to branch ch-change-static-links-162657578

- Open the templates directory and notice that there are no urls refering to static pages
